### PR TITLE
go/wallet_client: add flag to set affinity

### PR DESCRIPTION
**Pull request by menghanl**
If `--affinity=true`, RPCs will be sent to metadata `{session_id:1234}`.

As done in CPP #54, Java #61

### CODE REVIEW

File: go/wallet_client/main.go [hunk 1]
```diff
@@ -115,7 +117,13 @@ flags
 	}
 }
 
-// createMetaData creates the metadata for an outgoing request based on the user.
+const (
+	affinityMDKey   = "session_id"
+	affinityMDValue = "1234"
+)
```
Comment (easwars) [inline go/wallet_client/main.go:123]: Would we ever want these to be flags instead of consts?

**Review by easwars — APPROVED** [review]

File: go/wallet_client/main.go (hunk 1 above)
Comment (menghanl) [reply go/wallet_client/main.go:123]: The key would just be a const I believe.
The value we may want to change later. But we don't need it now.

Comment (easwars) [reply go/wallet_client/main.go:123]: Ack.

**Review by donnadionne — APPROVED** [review]

**Comment by dapengzhang0** [comment]
Backport to v1.1.x branch?

**Pull request merged**